Deploy/UtilityScripts/CpqApiHelper.py

```diff
@@ -115,10 +116,79 @@ def deleteGlobalScript(self, id):
         )
         return response
 
+    # Custom Table APIS
+
+    def getAllCustomTables(self):
+        print("121")
+        headers = self.getCTHeaderJwt()
+        headers["Content-Type"] = "application/json"
```

Comment:
do we need another getJTW method? beyond getHeaderJWT() and getHeaderBearer()

Let's maybe add the {"accept": "*/*"} to the json package on both?  What do you think?